src/components/PeoplePage/PeoplePage.tsx

```diff
@@ -0,0 +1,113 @@
+import React, { useState, useEffect } from 'react';
+import { useParams } from 'react-router-dom';
+import cn from 'classnames';
+import { Person } from '../../types';
+import { Loader } from '../Loader/Loader';
+import { getPeople } from '../../api';
+import { PersonLink } from '../PersonLink/PersonLink';
+
+export const PeoplePage: React.FC = () => {
+  const { slug = '' } = useParams();
+  const [people, setPeople] = useState<Person[]>([]);
+  const [isPeopleLoading, setIsPeopleLoading] = useState(false);
+  const [isPeopleError, setIsPeopleError] = useState(false);
+
+  const isSelected = (person: Person) => person.slug === slug;
+
+  useEffect(() => {
+    setIsPeopleLoading(true);
+    getPeople()
+      .then(setPeople)
+      .catch(() => setIsPeopleError(true))
+      .finally(() => setIsPeopleLoading(false));
+  }, []);
+
+  const findParent = (name: string) => {
+    const parent = people.find(pers => pers.name === name);
+
+    return parent
+      ? <PersonLink person={parent} />
+      : name;
+  };
+
+  const tableHeaders = [
+    { id: 1, title: 'Name' },
+    { id: 2, title: 'Sex' },
+    { id: 3, title: 'Born' },
+    { id: 4, title: 'Died' },
+    { id: 5, title: 'Mother' },
+    { id: 6, title: 'Father' },
+  ];
+
+  return (
+    <>
+      <h1 className="title">People Page</h1>
+      <div className="block">
+        <div className="box table-container">
+
+          {isPeopleLoading && (
+            <Loader />
+          )}
+          {!people.length && !isPeopleLoading && (
+            <p data-cy="noPeopleMessage">
+              There are no people on the server
+            </p>
+          )}
```

Comment:
This section is visible for a millisecond, fix your condition

https://user-images.githubusercontent.com/57747481/216374577-bdf9bccb-bea4-4b03-9233-7163d9d2ab2c.mov